### PR TITLE
[plugin.video.bfi] 1.1.0

### PR DESCRIPTION
**Pull request by FraserChapman**
### Description

Python 3 compatible 
Removed all hardcoded special URIs

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by FraserChapman** [comment]
Forgot to remove the .gitignore :)

File: plugin.video.bfi/resources/lib/kodiutils.py [hunk 1]
```diff
@@ -28,79 +28,64 @@ def art(domain, image):
...
 
 
 def set_setting(setting, value):
     ADDON.setSetting(setting, str(value))
 
 
 def get_setting_as_bool(setting):
-    return get_setting(setting).lower() == "true"
+    # type (str) -> bool
+    return ADDON.getSettingBool(setting)
```
Comment (Rechi) [inline plugin.video.bfi/resources/lib/kodiutils.py]: This is v18 API.

File: plugin.video.bfi/resources/lib/kodiutils.py [hunk 2]
```diff
@@ -28,79 +28,64 @@ def art(domain, image):
...
 
 
 def get_setting_as_float(setting):
+    # type (str) -> float
     try:
-        return float(get_setting(setting))
+        return ADDON.getSettingNumber(setting)
```
Comment (Rechi) [inline plugin.video.bfi/resources/lib/kodiutils.py]: This is v18 API.

File: plugin.video.bfi/resources/lib/kodiutils.py [hunk 3]
```diff
@@ -28,79 +28,64 @@ def art(domain, image):
...
     except ValueError:
         return 0
 
 
 def get_setting_as_int(setting):
+    # type (str) -> int
     try:
-        return int(get_setting_as_float(setting))
+        return ADDON.getSettingInt(setting)
```
Comment (Rechi) [inline plugin.video.bfi/resources/lib/kodiutils.py]: This is v18 API.

File: plugin.video.bfi/resources/lib/kodiutils.py (hunk 1 above)
Comment (FraserChapman) [reply plugin.video.bfi/resources/lib/kodiutils.py]: done

**Pull request merged**